scripts/robot.py

```diff
@@ -0,0 +1,34 @@
+from compas_robots import RobotModel
+from compas_robots.resources import GithubPackageMeshLoader
+from compas_robots.viewer.scene.robotmodelobject import RobotModelObject
+from compas_viewer.components import Slider
+from compas_viewer import Viewer
+
+viewer = Viewer(rendermode="lighted")
+
+
+github = GithubPackageMeshLoader("ros-industrial/abb", "abb_irb6600_support", "kinetic-devel")
+model = RobotModel.from_urdf_file(github.load_urdf("irb6640.urdf"))
+model.load_geometry(github)
+
+configuration = model.zero_configuration()
+robot_object: RobotModelObject = viewer.scene.add(model, show_lines=True, show_points=False, configuration=configuration)  # type: ignore
```

Comment:
remove `show_lines` and `show_points`

Reply:
do you know, in this class,robot_object: RobotModelObject = viewer.scene.add(model,.....   How to assemble a robot tool

Reply:
robot.attach_tool(tool,group,ee_link_name)
robot_object.attach_tool_model(tool.tool_model)
robot_object.update(config)
viewer.renderer.update()
。。。The above does not work。。。。